lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -46,8 +60,8 @@ defmodule Kernel.LexicalTracker do
 
   # Starts the tracker and returns its pid.
   @doc false
-  def start_link(dest) do
-    :gen_server.start_link(__MODULE__, dest, [])
+  def start_link(dest, parent) do
```

Comment:
Derp. You're totally right, I was just working within the confines of `each_module` and missed this case. I'll back all the "parent" stuff out.